fitbenchmarking/fitbenchmarking
# 754 mantid support for bounds

#### Description of Work

Fixes #754

#### Testing Instructions

1. Run Fitbenchmarking with Mantid minimizers and check that constraints are handled correctly for bounded problems
2. Check that results are consistent with master when using problems without bounds

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
